### PR TITLE
added sudhanshuedoc's blog

### PR DESCRIPTION
**Pull request by Sudhanshuedoc**
Added my photo and listed my blog in the repo.

[Live Link](https://sudhanshuedoc.github.io/gci17.fossasia.org/)

### CODE REVIEW

**Comment by abishekvashok** [comment]
@Sudhanshuedoc pls resize your image to 240x240px

File: index.html [hunk 1]
```diff
@@ -389,6 +389,18 @@ <h4 class="uppercase mb16">Blog</h4>
 					</div>
 				</div>
 				<div class="row">
+					<div class="col-md-3 col-sm-6">
+						<a href="https://participationgcin.wordpress.com/" target="_blank">
+							<div class="image-tile outer-title text-center">
+								<img alt="Pic" src="img/students/sudhanshuedoc.jpg" height="240" width="240">
```
Comment (abishekvashok) [inline index.html]: Resize your image

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Sudhanshuedoc) [reply index.html]: Above u said resize ur image which image should I resize Introduction part one or homepage part of my blog
??????

Comment (abishekvashok) [reply index.html]: @Sudhanshuedoc pls resize img/students/sudhanshuedoc.jpg, the image added with this PR.

Comment (Sudhanshuedoc) [reply index.html]: Ok sir got u. Sir can I do that tomorrow

Comment (Sudhanshuedoc) [reply index.html]: ????

Comment (Sudhanshuedoc) [reply index.html]: Actually my pc stopped working so am replying u with my phone so can I do that tomorrow?????

Comment (abishekvashok) [reply index.html]: @Sudhanshuedoc take your own time,
no need to call me sir, I am fellow student like you
take your own time, we will be here itself, it's open source! Enjoy

Comment (Sudhanshuedoc) [reply index.html]: Am out with my deadline that will not not gonna affect my project na????

Comment (Sudhanshuedoc) [reply index.html]: ???

Comment (abishekvashok) [reply index.html]: @Sudhanshuedoc you have I guess 7 days time to work on this issue, if you request mentors will give you more time, if you fail also, your task will get rejected. Usually, the extension time will be enough for you. If you can't get it in extension time also, don't worry find a new task to work on, this PR will remain here. You have all the time in the world to make works here, (7 days for task). :)

**Comment by NaviRocker** [comment]
Please Resolve the Conflicts

**Review by isuruAb — COMMENTED** [review]
@Sudhanshuedoc  please resolve conflicts and resize image to 240x240px

**Comment by ParthS007** [comment]
@Sudhanshuedoc please resolve conflicts and update your pr with the suggested changes

**Review by Sudhanshuedoc — COMMENTED** [review]
in coding only na i have to put my image size height:240px;
and 
width:240px;
??????

**Review by ParthS007 — COMMENTED** [review]
@Sudhanshuedoc You have to add an image in ` img/students/` which have dimmension of `240 * 240` otherwise Travis will not pass.

**Review by dilpreetsio — COMMENTED** [review]
@Sudhanshuedoc Please resolve the conflicts & it looks good to merge.

**Comment by Sudhanshuedoc** [comment]
geekyd  Please resolve the conflicts & it looks good to merge. i didn't got u

**Comment by Sudhanshuedoc** [comment]
ok i would resolve the conflict

**Comment by Sudhanshuedoc** [comment]
Resolved Merge Conflicts and added my blog data in relative place. :)

File: _data/blogs.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
-﻿- url: http://gci2017fossasia.blogspot.in/2017/11/how-to-clone-biggest-repositories.html
+﻿- url: https://participationgcin.wordpress.com/
+  title: GET LOST IN THE DEPTH OF LEARNING WITH GOOGLE CODE-IN
+  img: sudhanshublog.png
```
Comment (harshit98) [inline _data/blogs.yml:3]: Use your image `subdanshuedoc.jpg`. Remove `sudhanshublog.png`

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@Sudhanshuedoc Please resolve conflicts and do suggested changes 👍

**Comment by supunTE** [comment]
Resolve conflicts

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Resolve Conflicts @Sudhanshuedoc :+1:

**Comment by harshit98** [comment]
@Sudhanshuedoc Kindly do the changes which I have suggested.

I'm closing this PR due to no activity from a long time. I would suggest you to open a fresh PR if you're unable to resolve conflicts.

**Pull request closed without merging**